contract/models/contract_line.py

```diff
@@ -42,6 +42,11 @@ class ContractLine(models.Model):
     )
     product_id = fields.Many2one(index=True)
 
+    @api.depends("name", "date_start")
+    def _compute_display_name(self):
+        for rec in self:
+            rec.display_name = self.env._("%s - %s") % (rec.date_start, rec.name)
```

Comment:
This shouldn't be translatable AFAIK.

Reply:
This has high chances of being broken by incorrect translations (someone removes a marker), and anyway, you can even not customize the order of both parameter, as they are not keyword placeholders. Please remove it.